variables.tf

```diff
@@ -71,6 +71,18 @@ variable "timeout" {
   description = "The maximum amount of time in seconds to allow the job to run."
 }
 
+variable "parallelism" {
+  default     = ""
+  type        = string
+  description = "Specifies the maximum desired number of tasks the execution should run at given time. Must be <= task_count."
+}
+
+variable "task_count" {
+  default     = "1"
```

Comment:
wonder if this should be empty so existing behaviour is preserved?  I was looking at https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_job and it says that `task_count` is optional?